index.html

```diff
@@ -0,0 +1,189 @@
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+              <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+              <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+              <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+              <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+              <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                 
+              </div>
+              </div>
+               <div class="row  mt-3">
+                <div class="col-sm-6">
+                    <img src="images/image-1.jpg" class="img-fluid" alt="Responsive image">
+              </div>
+                <div class="col-sm-6 ">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
```

Comment:
Missing comment section.
Try using a grid layout to incorporate comment section